### PR TITLE
Roles within Zowe

### PR DESCRIPTION
**Pull request by balhar-jakub**
Linked to: #922 

Signed-off-by: Jakub Balhar <jakub.balhar@broadcom.com>

### CODE REVIEW

File: Technical-Steering-Committee/contributing.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+- Every action not already listed, such as but not limited to
+- Adding and removing users to the roles
+- Changing branch permissions
+- Changing github hooks
+- Changing repo status
+
+### Zowe administrators
+
+Selected by TSC. Needs to be voted to the position. The zowe wide administrator will have the Administrator privileges for all Zowe repositories.
```
Comment (jackjia-ibm) [inline Technical-Steering-Committee/contributing.md:65]: Not sure what's the common practice here, like how many owners are suggested for a github org. To add people to the org, afaik it needs owner permission. If we automatically promote TSC members as Zowe administrators, then each squad will have a representative as zowe github owner, and will be able to add members by themselves.

Just the squad representative should be careful not to touch other squads permissions. I believe TSC members will understand well and behave as expected.

File: Technical-Steering-Committee/contributing.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+- Can review PRs
+- Edit issues and PRs (labels, closing)
+
+General expectations:
+
+- Behavior in line with the Code of Conduct
+- Communication around the created issues/PRs with the squad
+
+### Committer
```
[the remark this answers is not in the document]
Comment (balhar-jakub) [reply Technical-Steering-Committee/contributing.md:39]: Passive committer

File: Technical-Steering-Committee/contributing.md (hunk 1 above)
Comment (balhar-jakub) [reply Technical-Steering-Committee/contributing.md:75]: Where do we keep the information?
Who are the people?
How do we vote and for how long?
Review the current owners for Zowe?

File: Technical-Steering-Committee/contributing.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+# Contributing and roles
+
+This document explains roles within Zowe and the ways and requirements for contribution.  
+
+The goal of the document is to outline key roles within Zowe with the guidance for the squads around the privileges associated with the roles. Each squad can update the privileges as needed. Apart from the privileges the document also outlines general expectations for the roles if there are any. The exact structure and rules if changed will be in the CONTRIBUTING.md in the repositories belonging to the squad. 
+
+The main source of the truth for who is committer and administrator for the project will be [COMMITTERS.md](https://github.com/zowe/community/blob/master/COMMITTERS.md).The roles will be reflected in Teams on the Zowe GitHub ([Github Teams](https://github.com/orgs/zowe/teams)) for squads where it's appropriate. Each Squad will have a Committers, Contributors and Administrators teams.
```
Comment (balhar-jakub) [inline Technical-Steering-Committee/contributing.md:7]: Use the GitHub teams for the COMMITTERS as the baseline.

**Review by MarkAckert — APPROVED** [review]
We are able to implement the roles defined in this PR 👍

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**